### PR TITLE
Added fix for stabilizing sts test

### PR DESCRIPTION
**Pull request by vivekanandan-k-rh**
Signed-off-by: Vivekanandan K <vivk@redhat.com>

Added fix for stabilizing sts test

Logs of three consecutive passed runs: http://pastebin.test.redhat.com/1067609

### CODE REVIEW

File: rgw/v2/tests/s3_swift/test_sts_using_boto_server_side_copy.py [hunk 1]
```diff
@@ -201,10 +202,16 @@ def test_exec(config):
...
         log.info(f"copy object name: {copy_object_name}")
-        bucket2.copy(copy_source, copy_object_name)
+        rgw_s3_client.copy_object(
+            Bucket=bucket2.name,
+            Key=copy_object_name,
+            CopySource={
+                "Bucket": bucket1.name,
+                "Key": all_keys_in_buck1[0],
+            },
+        )
```
Comment (udaysk23) [inline rgw/v2/tests/s3_swift/test_sts_using_boto_server_side_copy.py:214]: Currently code is copying only one object from source to destination.
Can we copy all objects from source to destination

Comment (vivekanandan-k-rh) [reply rgw/v2/tests/s3_swift/test_sts_using_boto_server_side_copy.py:214]: Sure @udaysk23 . I'll update the code to copy all objects

Comment (vivekanandan-k-rh) [reply rgw/v2/tests/s3_swift/test_sts_using_boto_server_side_copy.py]: @udaysk23 , I have updated the code to copy all objects. Passed log: http://pastebin.test.redhat.com/1067624

**Review by udaysk23 — APPROVED** [review]
LGTM

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**